test/Reflection/FileReflectionTest.php

```diff
@@ -18,14 +18,6 @@
  */
 class FileReflectionTest extends \PHPUnit_Framework_TestCase
 {
-    public function testFileConstructor()
-    {
-        $filePath = __DIR__ . '/../../vendor/zendframework/zend-version/src/Version.php';
-        require_once $filePath;
-        $reflectionFile = new FileReflection($filePath);
-        $this->assertEquals(get_class($reflectionFile), 'Zend\Code\Reflection\FileReflection');
-    }
```

Comment:
I'd argue we still need the test, but should use a different class for it, perhaps just another class in the component.